core/sql/optimizer/RelExeUtil.cpp

```diff
@@ -3733,6 +3734,23 @@ RelExpr * DDLExpr::bindNode(BindWA *bindWA)
         return NULL;
       }
 
+      // Hive tables can only be specified as external and must be created
+      // with the FOR clause
+      if (qualObjName_.isHive()) 
+      {
+        if (createTableNode->isExternal())
+        {
+          externalTable = TRUE;
+          isHbase_ = TRUE;
```

Comment:
This is kind of tricky.  there is an isHive method in qualified name but I also needed to set isHbase_ in order for executeSeabaseDDL to work.  It this field is not set, then executeSeabaseDDL returns an unsupported error.